pkg/kubernetes/kubernetes.go

```diff
@@ -905,32 +910,57 @@ func (k *Kubernetes) ApplyObject(obj runtime.Object) error {
 }
 
 // InstallEverest downloads the manifest file and applies it against provisioned k8s cluster.
-func (k *Kubernetes) InstallEverest(ctx context.Context, namespace string) error {
-	data, err := scripts.Manifest()
+func (k *Kubernetes) InstallEverest(ctx context.Context, namespace string, version *goversion.Version) error {
+	if version == nil {
+		// No need to check the version if not provided.
+		k.l.Debug("No version provided to InstallEverest")
+		return nil
+	}
```

Comment:
Shouldn't this be considered an error?

Reply:
I had this in place for a purpose. I think it was due to upgrade since the code is so mixed up.
I can't remember why, so just returning error here and we'll handle it elsewhere once it pops up.